File: 06.streaming-os/StreamStudio/android/StreamStudioAndroid/app/src/main/java/com/lsam/streaming/streamstudio/network/Phase1Repository.java
package com.lsam.streaming.streamstudio.network;

public final class Phase1Repository {
    public String getDashboardRoute() {
        return ApiRoutes.DASHBOARD;
    }

    public String getProjectsRoute() {
        return ApiRoutes.PROJECTS;
    }

    public String getUploadQueueRoute() {
        return ApiRoutes.UPLOAD_QUEUE;
    }

    public String getPublishRequestRoute() {
        return ApiRoutes.PUBLISH_REQUEST;
    }
}

File: 06.streaming-os/StreamStudio/ios/StreamStudioiOS/Sources/Network/Phase1Repository.swift
import Foundation

struct Phase1Repository {
    func dashboardRoute() -> String {
        ApiRoutes.dashboard
    }

    func projectsRoute() -> String {
        ApiRoutes.projects
    }

    func uploadQueueRoute() -> String {
        ApiRoutes.uploadQueue
    }

    func publishRequestRoute() -> String {
        ApiRoutes.publishRequest
    }
}
